Modules/IO/BMP/test/itkBMPImageIOTestPalette.cxx

```diff
@@ -31,7 +31,9 @@ itkBMPImageIOTestPalette(int argc, char * argv[])
 {
   if (argc != 5)
   {
-    std::cerr << "Usage: " << argv[0] << " input"
+    std::cerr << "Missing parameters." << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv);
+    std::Cerr << " input"
```

Comment:
[CTest failure](https://open.cdash.org/viewBuildError.php?buildid=7347474) due to typo here:

> D:\a\1\s\Modules\IO\BMP\test\itkBMPImageIOTestPalette.cxx(36): error C2039: 'Cerr': is not a member of 'std'

Reply:
@tbirdso thanks for the review ! Fixed in 76ef46b.